NuRadioReco/modules/efieldToVoltageConverter.py

```diff
@@ -32,17 +32,15 @@ def __init__(self, log_level=logging.NOTSET):
         self.__debug = None
         self.__pre_pulse_time = None
         self.__post_pulse_time = None
-        self.__max_upsampling_factor = None
         self.__antenna_provider = None
+        
         self.logger = logging.getLogger('NuRadioReco.efieldToVoltageConverter')
         self.logger.setLevel(log_level)
         self.begin()
 
 
-    def begin(self, debug=False, uncertainty=None,
-              time_resolution=None,
-              pre_pulse_time=200 * units.ns,
-              post_pulse_time=200 * units.ns,
+    def begin(self, debug=False, uncertainty=None, time_resolution=None,
+              pre_pulse_time=200 * units.ns, post_pulse_time=400 * units.ns,
```

Comment:
I haven't checked, but is the increase in post_pulse_time actually necessary, or is it due to #530 ?

Reply:
maybe (as you mentioned he reported this) @EnriqueHuesca can answer this

Reply:
It is necessary - I tested that myself using some neutrino simulation. It might be that it became necessary when we changed `simulation.py` in the NuRadioMC 3.0 branch: Since we only pass trigger channels to this module, the variation in the pulse times at the antennas which affect the trace length before padding are smaller than when we passed a full rnog station with channels on helper strings to this module

Reply:
Okay, in which case the additional 200 ns is somewhat arbitrary, but considering we generally expect neutrino signals to arrive later in shallower channels and we normally use deeper channels to trigger, the asymmetry between pre and post pulse times at least makes sense.